src/dss/manifests.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dss
+
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mlflow-pvc
+  namespace: dss
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 1Gi
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
Putting this comment here, but can be the case for the rest of the YAML objects we apply:

Let's add common labels across all objects that are part of a "component". In cases of `Deployments` let's add those in both `.metadata` and in `.spec.selector.matchLabels`. This is also what we do in Kubeflow
https://github.com/kubeflow/manifests/blob/6cb9ff2fa2b25755eca5ce41c03d125aa8c8653b/contrib/kserve/kserve/kserve_kubeflow.yaml#L20523

I propose the following labels for the MLFlow components (Deployment, SVC)
- `app: dss-mlflow`
- `app.kubernetes.io/name: dss-mlflow`

Reply:
@misohu does have some already, although slightly different.  The `KubernetesResourceHandler` [applies labels](https://github.com/canonical/data-science-stack/pull/29/files#diff-601d3d8b56c9ee039324597dae9fbef6a90af671d96f5063b8f64ae1a840c7d7R79) to everything it deploys, although their design intent was more for letting the KRH easily clean everything up later.  I think it works for this purpose here too though.

I wonder if we should split these resources up into more than one yaml though.  I don't think the `dss` namespace should have a `mlflow` label.

Reply:
Actually the labels from the KubernetesResourceHandler wont quite match what you're asking, as they will not be in the Deployment's pod template

Reply:
@kimwnasptd any reason why we need both `app` and `app.kubernetes.io/name`?  This feels redundant

Reply:
Added some but slightly different than you requested @kimwnasptd.  wdyt?